docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -0,0 +1,107 @@
+[role="xpack"]
+[testenv="basic"]
+[[inference-processor]]
+=== {infer-cap} Processor
+
+Uses a pre-trained {dfanalytics} model to infer against the data that is being 
+ingested in the pipeline.
+
+
+[[inference-options]]
+.{infer-cap} Options
+[options="header"]
+|======
+| Name               | Required  | Default                        | Description
+| `model_id   `      | yes       | -                              | (String) The ID of the model to load and infer against.
+| `target_field`     | no        | `ml.inference.<processor_tag>` | (String) The field name to which to write the results objects.
```

Comment:
I like the way this is worded in the enrich processor.  Can we do something similar? For example:
```suggestion
| `target_field`     | no        | `ml.inference.<processor_tag>` | (String) Field added to incoming documents to contain results objects.
```